website/blog/2021-02-22-jest-27.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Jest 27: Leaving the past behind ⏩
+author: Tim Seckinger
+authorURL: https://github.com/jeysal
+authorImageURL: https://avatars.githubusercontent.com/u/16069751
+---
+
+In the [Jest 26 blog post](https://jestjs.io/blog/2020/05/05/jest-26) almost a year ago, we announced that after two major releases with few breaking changes, Jest 27 would flip some switches to set better defaults for projects that are new or can migrate smoothly. This will then also give us the opportunity in Jest 28 to remove some packages from the default distribution of Jest and publish them as separately installable and pluggable modules instead, so that everyone on the new defaults can benefit from a smaller install size.
+
+We will explain those changes of defaults and other notable breaking changes in this post, but first, let's get into some exciting new features shipping in Jest 27!
```

Comment:
I recommend swapping this. Talk about the new defaults first (that's the intent of the blog post) and features second.

Reply:
Hmm, problem I see with that is it would split breaking changes from sections 2,3,4 to sections 1,3,4. Also think it's good to show off some nice free stuff first, because to some people breaking changes will always look like an annoyance even if they are well-reasoned.